src/scripts/run.sh

```diff
@@ -5,73 +5,74 @@ airflow initdb  # initdb is not destructive, so can be re-run at startup
 python scripts/mkvars.py
 
 # creating an admin and regular users (nessacary when using RABC=True in the airflow.cnf)
-airflow create_user -r Admin -u admin -e admin@example.com -f admin -l admin -p ${AIRFLOW_USER_ADMIN_PASSWD}
-airflow create_user -r User -u dataservices -e dataservices@example.com -f dataservices -l dataservices -p ${AIRFLOW_USER_DATASERVICES_PASSWD}
-airflow create_user -r User -u team_ruimte -e team_ruimte@example.com -f team_ruimte -l team_ruimte -p ${AIRFLOW_USER_TEAM_RUIMTE_PASSWD}
+airflow users create -r Admin -u admin -e admin@example.com -f admin -l admin -p ${AIRFLOW_USER_ADMIN_PASSWD:-admin}
+
+airflow users create -r User -u dataservices -e dataservices@example.com -f dataservices -l dataservices -p ${AIRFLOW_USER_DATASERVICES_PASSWD:-dataservices}
+airflow users create -r User -u team_ruimte -e team_ruimte@example.com -f team_ruimte -l team_ruimte -p ${AIRFLOW_USER_TEAM_RUIMTE_PASSWD:-team_ruimte}
 
 # Airflow does not support slack connection config through environment var
 # So we (re-)create the slack connection on startup.
-airflow connections --delete --conn_id slack
-airflow connections --add  --conn_id slack --conn_host $SLACK_WEBHOOK_HOST \
-    --conn_password "/$SLACK_WEBHOOK" --conn_type http
+airflow connections delete slack
+airflow connections add slack --conn-host $SLACK_WEBHOOK_HOST \
+    --conn-password "/$SLACK_WEBHOOK" --conn-type http
 
-airflow connections --delete --conn_id geozet_conn_id
-airflow connections --add  --conn_id geozet_conn_id --conn_host http://geozet.koop.overheid.nl \
-    --conn_type http
+airflow connections delete geozet_conn_id
```

Comment:
Ik merk bij opstarten airflow dat het aanmaken van de connecties op deze manier relatief traag is. Het zou evt. vervangen kunnen worden door een opzet obv. environment variabelen zoals bijv.: `AIRFLOW_CONN_GOB_GRAPHQL: "https://acc.api.data.amsterdam.nl/https"`
Dit lukte aanvankelijk niet voor https connecties, maar met het trucje zoals hierboven  (`/https` toevoegen) is dat toch mogelijk. Wellicht voert het voor nu te ver om de connecties te vervangen, maar kunnen we nieuwe connecties het beste via env. vars doen en de huidige `cli-based` connecties langzamerhand vervangen.

Reply:
Goede, maar ik weet niet of nu is het tijd om alle connecties te vervangen.

Reply:
Nee, voor nu zou ik het zo laten. Alleen nieuwe connecties niet meer op deze manier, maar als env var.

Reply:
Ik heb een warning toegevoegt. https://github.com/Amsterdam/dataservices-airflow/pull/176/commits/f13b257fa12f8aa76423dd8bffa9e860f3b9d870